src/app/app.component.ts

```diff
@@ -181,6 +190,12 @@ export class MyApp {
 
     });
 
+    this.localStorage.getLocalStorage(storageKeys.staticLinks).then(success => {
+     console.log(success)
+     if(success['tutorial-video'].metaInformation.videos.length==0)this.allPages[this.findIndex("tutorialvideo")]['show']=false
```

Comment:
will this work??

Reply:
yes i tested it with length ==1 as we already have one video so it was hiding in side menu ..it should work with length ==0  also and should hide  from side menu . tried after logout also. it was working